i18n/translations/fr/form-builder.json

```diff
@@ -530,7 +530,8 @@
   "gcFormsStart": "Créer",
   "gcFormsTest": "Tester",
   "gcFormsTranslate": "Traductions",
-  "hasFileInputAndApiDelivery": "[FR] Has FileInput and API delivery enabled",
+  "hasFileInputAndApiDelivery": "API delivery enabled [FR]",
```

Comment:
@anikbrazeau 

This translation was missing but I also updated the code to add a note

I think that works better - thoughts ?

<img width="431" alt="Screenshot 2025-06-11 at 7 04 51 AM" src="https://github.com/user-attachments/assets/4769fc85-1bfb-4dc0-a466-3e4a53f68c32" />